OracleDatabase/SingleInstance/dockerfiles/23.4.0/Containerfile.free

```diff
@@ -98,21 +99,21 @@ FROM base
 USER oracle
 COPY --chown=oracle:oinstall --from=builder $ORACLE_BASE $ORACLE_BASE
 COPY --chown=oracle:oinstall --from=builder /etc/oratab /etc/oratab
-COPY --from=builder /etc/init.d/oracle-free-23c /etc/init.d/oracle-free-23c
-COPY --from=builder /usr/share/doc/oracle-free-23c/LICENSE /usr/share/doc/oracle-free-23c/LICENSE
+COPY --from=builder /etc/init.d/oracle-free-23ai /etc/init.d/oracle-free-23ai
+COPY --from=builder /usr/share/doc/oracle-free-23ai/LICENSE /usr/share/doc/oracle-free-23ai/LICENSE
 
 USER root
 # To enforce ORACLE_SID as FREE, adding it to .bashrc
-RUN $ORACLE_BASE/oraInventory/orainstRoot.sh && \
-    $ORACLE_HOME/root.sh && \
-    echo 'export ORACLE_SID=FREE' >> /home/oracle/.bashrc && \
+RUN $ORACLE_BASE/oraInventory/orainstRoot.sh &&
+    $ORACLE_HOME/root.sh &&
+    echo 'export ORACLE_SID=FREE' >>/home/oracle/.bashrc &&
     chown oracle.oinstall /home/oracle/.bashrc
```

Comment:
```suggestion
RUN $ORACLE_BASE/oraInventory/orainstRoot.sh && \
    $ORACLE_HOME/root.sh && \
    echo 'export ORACLE_SID=FREE' >>/home/oracle/.bashrc && \
    chown oracle.oinstall /home/oracle/.bashrc
```

Reply:
It got removed by local shell script formatting extn. Fixed now. Thanks